charts/splunk-connect-for-snmp/values.yaml

```diff
@@ -2,36 +2,59 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
+################################################################################
+# SC4SNMP image settings
+################################################################################
 
 image:
+  # The registry and name of the SC4SNMP image to pull
   repository: ghcr.io/splunk/splunk-connect-for-snmp/container
+  # The policy that specifies when the user wants the SC4SNMP images to be pulled
   pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
   tag: ""
 
+# Secrets to attach to the respective serviceaccount to pull docker images
 imagePullSecrets: []
 
-splunk:
-  enabled: true
 
-inventory:
-  nameOverride: ""
-  fullnameOverride: ""
+################################################################################
+# Splunk Data-to-Everything Platform configuration
+################################################################################
 
-  podAnnotations: {}
+splunk:
+  # Enables sending data to Splunk
+  enabled: true
+  # the protocol of the HEC endpoint: https or http
+  protocol: ""
+  # the port of the HEC endpoint
+  port: "8088"
+  # IP address or a domain name of a Splunk instance to send data to.
+  host: ""
+  # the protocol, host and port given here makes up to a HEC endpoint
+  # according to the pattern: {{protocol}}://{{host}}:{{port}}/services/collector
+  # for ex. https://splunk-endpoint:8088/services/collector
+
+  # Required for Splunk Enterprise/Cloud (if `enabled` is set to true). Splunk
+  # HTTP Event Collector token.
+  token: 00000000-0000-0000-0000-000000000000
+  # Whether to skip checking the certificate of the HEC endpoint when sending
+  # data over HTTPS.
+  insecureSSL: "true"
```

Comment:
If `insecureSSL` is false, does it have any mechanism to provide root ca?

Reply:
No, we don't have a mechinsm to provide own root ca yet. This option makes SC4SNMP skip the verification step and proceed without checking.